pkg/mixin/helpers.go

```diff
@@ -5,42 +5,49 @@ import (
 	"io/ioutil"
 
 	"get.porter.sh/porter/pkg/context"
+	"get.porter.sh/porter/pkg/pkgmgmt"
 )
 
+var _ pkgmgmt.PackageManager = &TestMixinProvider{}
+
 // TestMixinProvider helps us test Porter.Mixins in our unit tests without actually hitting any real mixins on the file system.
 type TestMixinProvider struct {
-	RunAssertions []func(mixinCxt *context.Context, mixinName string, commandOpts CommandOptions)
+	RunAssertions []func(mixinCxt *context.Context, mixinName string, commandOpts pkgmgmt.CommandOptions)
 }
 
-func (p *TestMixinProvider) List() ([]Metadata, error) {
-	mixins := []Metadata{
+func (p *TestMixinProvider) List() ([]pkgmgmt.Metadata, error) {
```

Comment:
Do we want to change the name from `TestMixinProvider` to just, say, `TestMixin`?  Noticed the 'real' former `MixinProvider` looks to be changed to just `Mixin`...

Reply:
I believe this name is more consistent with our other test providers naming `Test+Struct|InterfaceName`, and follows what @dev-drprasad did in their PR:

Here's some of the other test stubs we have for reference
https://github.com/deislabs/porter/blob/b27f4a04f9a143c0c853ea37ba71b07d597bad6b/pkg/porter/helpers.go#L44-L55